tests/test_optimizer.py

```diff
@@ -917,6 +917,7 @@ def test_constrained_AL_lsq():
 
 @pytest.mark.slow
 @pytest.mark.regression
+@pytest.mark.xfail
 def test_constrained_AL_scalar():
     """Tests that the augmented Lagrangian constrained optimizer does something."""
     eq = desc.examples.get("SOLOVEV")
```

Comment:
Marked this as okay to fail because the optimization stalls at ctol = 1e-2 on the GitHub CI. Works fine on my computer. This happened in an earlier pull request. It was somewhat weird then that increasing precision in the integration here https://github.com/PlasmaControl/DESC/pull/501/commits/9ac5b656f095fae6585f29528839ce9e948a28ca allowed the GitHub CI to pass, when the actual solution then was to set up the constraint tolerance properly. We already set `jax.numpy` precision to 64bit in `desc.backend`.

Reply:
I was having issues with this test on another branch as well, it seems like it's probably pretty sensitive to small numerical differences, I'll work on a proper fix but `xfail` is fine for now.

Reply:
I don't have any justification for this, but I think some of the sensitivity may come from the solved solovev equilibrium. Maybe try to switch it to dshape if that's an option